project/plugins.sbt

```diff
@@ -22,6 +22,6 @@ addSbtPlugin("com.typesafe.sbt" % "sbt-uglify" % "2.0.0")
 
 addSbtPlugin("com.typesafe.sbt" % "sbt-digest" % "1.1.4")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-accessibility-linter" % "0.37.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-accessibility-linter" % "0.39.0")
 
-addSbtPlugin("io.github.irundaia" % "sbt-sassify" % "1.5.2")
\ No newline at end of file
+addSbtPlugin("io.github.irundaia" % "sbt-sassify" % "1.5.2")
```

Comment:
why is this showing as removed and added 🤔

Reply:
I think this is one of those github quirks where you add a new line at the end of the file